turbo.json

```diff
@@ -9,15 +9,38 @@
     "lint": {},
     "test": {},
     "build": {
-      "dependsOn": ["audit", "lint", "test", "^build"],
-      "outputs": ["dist/**", ".next/**", ".storybook/**"]
+      "dependsOn": [
+        "audit",
+        "lint",
+        "test",
+        "^build"
+      ],
+      "outputs": [
+        "dist/**",
+        ".next/**",
+        ".storybook/**"
+      ]
     },
     "deploy": {
-      "dependsOn": ["^build"],
-      "env": ["CDK_DEFAULT_ACCOUNT", "CDK_DEFAULT_REGION", "DOMAIN_NAME"]
+      "dependsOn": [
+        "^build"
+      ],
+      "env": [
+        "CDK_DEFAULT_ACCOUNT",
+        "CDK_DEFAULT_REGION",
+        "DOMAIN_NAME"
+      ]
     },
     "destroy": {
-      "env": ["CDK_DEFAULT_ACCOUNT", "CDK_DEFAULT_REGION"]
+      "env": [
+        "CDK_DEFAULT_ACCOUNT",
+        "CDK_DEFAULT_REGION"
+      ]
+    },
+    "react-pagination#build": {
+      "dependsOn": [
+        "essentials#build"
+      ]
     }
```

Comment:
Please remove this and instead mark `essentials` as a dependency in `react-pagination`. That way, the `^build` in the `dependsOn` will take care of this automatically, because `^build` will make sure that `build` is ran on any dependencies that have a build script.